hera_mc/cm_hookup.py

```diff
@@ -279,6 +279,32 @@ def get_correlator_input_from_hookup(self, hookup_dict):
                     corr_input[j] = p[-1].downstream_part
         return corr_input
 
+    def get_pam_from_hookup(self, hookup_dict, pam_name='post-amp'):
+        """
+        Retrieve the PAM connections from a hookup dictionary.
+        No rationalization is provided.
```

Comment:
I don't understand this sentence.

Reply:
Me neither.

Reply:
I see, they are having fun with the comment in the previous method.